disperser/disperser.go

```diff
@@ -165,7 +165,8 @@ type BlobStore interface {
 	// GetBlobMetadata returns a blob metadata given a metadata key
 	GetBlobMetadata(ctx context.Context, blobKey BlobKey) (*BlobMetadata, error)
 	// HandleBlobFailure handles a blob failure by either incrementing the retry count or marking the blob as failed
-	HandleBlobFailure(ctx context.Context, metadata *BlobMetadata, maxRetry uint) error
+	// Returns a boolean indicating whether the blob should be retried and an error
```

Comment:
nit: It may be fine to omit subject but may need to keep it consistent with methods above

Reply:
Can you clarify the suggestion here? This is consistent with methods above L141, L144

Reply:
I see, there is "HandleBlobFailure handles" at L167